include/aikido/perception/AssetDatabase.hpp

```diff
@@ -34,34 +34,34 @@ namespace perception {
 ///  }
 /// \endcode
 
-class ObjectDatabase
+class AssetDatabase
 {
 public:
-  /// Construct a \c ObjectDatabase that uses \c ResourceRetriever to
+  /// Construct a \c AssetDatabase that uses \c ResourceRetriever to
   /// load configuration data from a JSON file at URI \c configDataURI.
   /// \param[in] resourceRetriever The pointer to obtain the configuration file
   /// \param[in] configDataURI The URI for the configuration information file
-  ObjectDatabase(
+  AssetDatabase(
       const dart::common::ResourceRetrieverPtr& resourceRetriever,
       const dart::common::Uri& configDataURI);
 
-  virtual ~ObjectDatabase() = default;
+  virtual ~AssetDatabase() = default;
 
   /// Get the object name, resource, and offset from database by objectKey
-  /// \param[in]  objectKey The key (string) of an object in ObjectDatabase
-  /// \param[out] objectName The retrieved object name from ObjectDatabase
-  /// \param[out] objectResource The retrieved uri of the object
-  /// \param[out] objectOffset The retrieved offset matrix of the object
+  /// \param[in]  assetKey The key (string) of an object in AssetDatabase
+  /// \param[out] assetName The retrieved object name from AssetDatabase
+  /// \param[out] assetResource The retrieved uri of the object
+  /// \param[out] assetOffset The retrieved offset matrix of the object
   ///     i.e. the offset between a tag and the actual origin of an object
```

Comment:
Nit: `i.e.` should probably be `e.g.` here since the "tag" refers to AprilTag in this context.